gcp/appengine/blog/content/posts/almalinux-and-rocky-linux-join-osv.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "AlmaLinux and Rocky Linux join OSV"
+date: 2023-05-08T16:00:00Z
+draft: false
+author: OSV Team
+---
+
+Two new Linux distributions have been added to the OSV database. With the addition of [AlmaLinux](https://almalinux.org/) and [Rocky Linux](https://rockylinux.org/), the OSV database is now made up of advisories from 18 sources, including language ecosystems and Linux distributions.
+<!--more-->
+
+Alma Linux is an open source, community owned and governed, enterprise linux distro focused on long-term stability and providing a robust production-grade platform. Rocky Linux is an open source enterprise operating system designed to be 100% bug-for-bug compatible with Red Hat Enterprise Linux. Both maintainer teams worked closely with the OSV team to bring their advisories into the database. We’d like to thank them for their hard work! We know our community will benefit from their data contributions. 
```

Comment:
We need 3 corrections here, please:
1. AlmaLinux -> AlmaLinux
2. linux -> Linux
3. Rocky Linux is an ... -> Both AlmaLinux and Rocky Linux are...

Reply:
Maybe "Alma Linux -> AlmaLinux"?

Reply:
Updated! Thank you both.